### DISCUSSION TITLE
Проверка типа блокировки ресурса

### DISCUSSION THREAD

**Discussion by alexdv124** [bol-van/zapret#287] [Q&A] [answered]
Коллеги, всем привет! Установил программу, на x86 пк, ОС Ubuntu server 24.04. ПК с двумя сетевухами, подключен между домашним роутером и медиаконвертером провайдера. основной функционал удалось запустить, ютуб работает, спасибо большое автору! Но некоторые сайты не открываются или открываются с перебоями (например facebook, instagram). blockcheck использовал, но результат по этим сайтам по тем стратегиям, которых дождался - unavailable. Вопрос: как достоверно определить, блокируется ли сайт по IP или с помощью dpi? Нужен какой то мануал, так как компетентности в linux не хватает

**Comment by bol-van** [comment]
блокчек проверяет порт блок и частичный ип блок
с портом все понятно. с частичным блоком как интерпретировать описано в readme

**Reply by bol-van** [reply, marked as answer]
если есть полный ип блок то естественно будет и порт блок

**Reply by alexdv124** [reply]
спасибо!

**Reply by alexdv124** [reply]
для информации: вот так выглядит вывод, когда порт не заблокирован, и следовательно, нет блокировки по ip(на примере meduza.io). 
![image](https://github.com/user-attachments/assets/7f5a24fa-753d-496b-bef1-109e342ec6c8)
а так, когда есть блокировка порта и, соответственно, ip
![image](https://github.com/user-attachments/assets/a4f4c299-65d7-4041-94c7-60198fd5db6c)
